diffkemp/building/cc_wrapper_gen.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import check_call
+import os
+
+
+def generate_cc_wrapper(rpython='rpython', builddir='build'):
+    wrapper_template = 'cc_wrapper_template'
+    compiled_wrapper_filename = os.path.join(os.getcwd(), builddir,
+                                             wrapper_template + '-c')
```

Comment:
I didn't like that it would recompile the wrapper each time `/tmp` is cleared, however a proper permanent solution would require moving the generation of the wrapper out of generate phase into DiffKemp's build system.

Now thinking about it again, some systems have `/tmp` mounted with `noexec`, which is a bigger issue. I'll move the wrapper to be built and installed by CMake together with SimpLL, that should work.